### PR TITLE
Added some additional labels on graves.

### PR DESCRIPTION
**Pull request by Daved27hundred**
#### Summary
Content "Graves may have ruder writing on them"

#### Purpose of change
Worldbuilding and storytelling purposes. What's more interesting than a grave with a heartfelt message? ~~One with an angry, spiteful message~~ one with a more bitter or negative message, raising questions about who was buried there. Were they an enemy or a friend turned enemy? Why did these people bury someone they weren't fond of? Did they feel guilty or still respect them enough as a human being to give them a proper burial?

#### Describe the solution

Added ~~28~~ 6 new snippets to the `grave_labels` file, which contains messages meant to appear on random graves in the fields.

#### Describe alternatives you've considered

Not adding them, adding less, or adding more (I am open to suggestions for snippets just comment them)

### CODE REVIEW

File: data/json/snippets/grave_labels.json [hunk 1]
```diff
@@ -31,7 +31,50 @@
...
+      "Waste of a perfectly good bullet",
+      "As much as the world is hell currently, your death made it that much closer to heaven",
+      "Shouldn't have stolen from us",
+      "Now I can rest easy",
+      "Now we can rest easy",
+      "<a crass drawing>",
+      "Even the zeds would make a better comrade than you",
+      "To think, you were the one to leave me for dead",
+      "Didn't have to end like this",
+      "Hope even the worms will touch your corpse",
```
Comment (MNG-cataclysm) [inline data/json/snippets/grave_labels.json:72]:
```suggestion
      "Hope even the won't will touch your corpse",
```
This seems to make better sense to me.

File: data/json/snippets/grave_labels.json [hunk 2]
```diff
@@ -31,7 +31,50 @@
...
+      "Here lies a traitor",
+      "Here lies a scoundrel",
+      "SCUM",
+      "LIAR",
+      "COWARD",
+      "I never liked you",
+      "We never liked you",
+      "Don't come back",
+      "Stay in the ground you sack of shit",
+      "<several slurs and insults from varying handwritings>",
```
Comment (MNG-cataclysm) [inline data/json/snippets/grave_labels.json:53]:
```suggestion
      "<several slurs and insults in various calligraphies>",
```

**Comment by Fris0uman** [comment]
Is that actually a thing? Who would pay to get a spiteful tpmbstone made? I'm no expert of cemeteries, especially in the US, but I've never seen something like this. If everybody hates you they don't take care of your funeral

**Comment by Qrox** [comment]
A tsundere, maybe.

Also, for graves made since the cataclysm, maybe someone thought leaving their enemy's corpse outside might risk some zombie necromancer to drop by and reanimate them so they decided to bury them anyway.

**Comment by Karol1223** [comment]
I'm with Fris on this one. This feels very Fallout-y. Not a huge fan.

File: data/json/snippets/grave_labels.json [hunk 3]
```diff
@@ -31,7 +31,50 @@
...
+      "Now we can rest easy",
+      "<a crass drawing>",
+      "Even the zeds would make a better comrade than you",
+      "To think, you were the one to leave me for dead",
+      "Didn't have to end like this",
+      "Hope even the worms will touch your corpse",
+      "Bitch",
+      "A burial was more than you deserved",
+      "Should've left your corpse where it lay",
+      "I wish you couldve seen reason",
```
Comment (Night-Pryanik) [inline data/json/snippets/grave_labels.json:76]:
```suggestion
      "I wish you could've seen reason",
```
Unless it's intended.

File: data/json/snippets/grave_labels.json [hunk 4]
```diff
@@ -31,7 +31,50 @@
...
+      "As much as the world is hell currently, your death made it that much closer to heaven",
+      "Shouldn't have stolen from us",
+      "Now I can rest easy",
+      "Now we can rest easy",
+      "<a crass drawing>",
+      "Even the zeds would make a better comrade than you",
+      "To think, you were the one to leave me for dead",
+      "Didn't have to end like this",
+      "Hope even the worms will touch your corpse",
+      "Bitch",
```
Comment (Night-Pryanik) [inline data/json/snippets/grave_labels.json:73]:
```suggestion
      <name_b>,
```
For various generic hostile/negative words. They will be in lower case, though.

**Comment by alp7292** [comment]
its fine to add them for me but i never saw someone vandalizing a grave in my life (apart from some popular controversial figures)  and they need to be rare. if its currently merged like this half of the grave labels will be negative and it wouldnt make sense 5 percent or lower would be good idea

**Comment by Daved27hundred** [comment]
> Is that actually a thing? Who would pay to get a spiteful tpmbstone made? I'm no expert of cemeteries, especially in the US, but I've never seen something like this. If everybody hates you they don't take care of your funeral

these are post-apocalypse graves exclusively, regular cemeteries aren't touched. 
I'll take Alps suggestion and limit them to a smaller number.

**Comment by Fris0uman** [comment]
almost everybody died in a few days, they were not carving mean tombstone they were digging mass graves.

**Comment by Daved27hundred** [comment]
these graves were likely dug by other, unseen groups of survivors like the other grave labels seem to imply. I get your point about them seeming too spiteful, so I cut the list down to the ones that were the least angry.

**Comment by Fris0uman** [comment]
Those labels appears on any grave, right? Even stone ones in cemeteries?

**Comment by Daved27hundred** [comment]
I don't believe so, but let me check.

**Comment by Daved27hundred** [comment]
Examining gravestones of several cemeteries ingame, none of them had any of the labels from the snippet file. I'm pretty sure that they exclusively appear on postapocalypse graves especially considering some of them mention zombies (eg. `may you never walk again, may i never kill you again`) maybe this should be made more apparent in the file to avoid confusion? like with a comment?

**Comment by Daved27hundred** [comment]
In that case I'm leaving a comment to point that out.

File: data/json/snippets/grave_labels.json [hunk 5]
```diff
@@ -31,7 +32,13 @@
       "I held you your whole life",
       "So small, so sweet, so soon",
       "Ours for a little while",
-      "You died for us, we'll live for you"
+      "You died for us, we'll live for you",
+      "Sorry, you had it coming",
+      "<nothing is written here>",
+      "Didn't have to end like this",
+      "Should've left your corpse where it lay",
```
Comment (Maleclypse) [inline data/json/snippets/grave_labels.json:39]:
```suggestion
      "Should've left your corpse where it lay, but I'm afraid it'd rise again and I'd have to put you down again",
```

File: data/json/snippets/grave_labels.json [hunk 6]
```diff
@@ -31,7 +32,13 @@
...
       "So small, so sweet, so soon",
       "Ours for a little while",
-      "You died for us, we'll live for you"
+      "You died for us, we'll live for you",
+      "Sorry, you had it coming",
+      "<nothing is written here>",
+      "Didn't have to end like this",
+      "Should've left your corpse where it lay, but I'm afraid it'd rise again and I'd have to put you down again",
+      "I wish you could've seen reason",
+      "A lesser man would've let you turn"
```
Comment (Maleclypse) [inline data/json/snippets/grave_labels.json:41]: The meta knowledge in this one has me shook! I love it, someone had in genre knowledge that zombie bites infect the living and turn them into zombies but that's not how zombies work in this world.  I adore this snippet for that.

File: data/json/snippets/grave_labels.json [hunk 7]
```diff
@@ -31,7 +32,13 @@
       "I held you your whole life",
       "So small, so sweet, so soon",
       "Ours for a little while",
-      "You died for us, we'll live for you"
+      "You died for us, we'll live for you",
+      "Sorry, you had it coming",
+      "<nothing is written here>",
+      "Didn't have to end like this",
+      "Should've left your corpse where it lay, but I'm afraid it'd rise again and I'd have to put you down again",
+      "I wish you could've seen reason",
```
Comment (Maleclypse) [inline data/json/snippets/grave_labels.json:40]: The idea that this is a loved one who turned feral is fantastic.

**Pull request merged**